go/common/grpc/grpc.go

```diff
@@ -298,8 +299,8 @@ type ServerConfig struct { // nolint: maligned
 	Port uint16
 	// Path is the path for the local server. Leave nil to create a TCP server.
 	Path string
-	// Certificate is the certificate used by the server. Should be nil for local servers.
-	Certificate *tls.Certificate
+	// Identity is the identity of the worker that's running the server.
+	Identity *identity.Identity
```

Comment:
This makes it kind of hard to reuse this code for other things (#2686).

Reply:
You can fill-in only the certificate field of the Identity struct and it will work the same way as before :)